platform/lf_nrf52_support.c

```diff
@@ -165,10 +191,6 @@ int lf_clock_gettime(instant_t* t) {
  * @return 0 on success, platform-specific error number otherwise.
  */
 int lf_mutex_lock(lf_mutex_t* mutex) {
```

Comment:
Shouldn't these mutex functions just be removed? The unthreaded runtime does not call them.

Reply:
removed